crates/astria-sequencer-utils/src/genesis_parser.rs

```diff
@@ -0,0 +1,68 @@
+use std::{
+    fs::File,
+    path::Path,
+};
+
+use anyhow::{
```

Comment:
Let's use `eyre` instead of `anyhow` (and `WrapErr` with `my_result.wrap_err` instead of `Context` with `my_result.context`).

Reply:
converted from anyhow to eyre